KotlinCMakeModule/CMakeDetermineKotlinCompiler.cmake

```diff
@@ -0,0 +1,38 @@
+if (NOT CMAKE_Kotlin_COMPILER)
+    #TODO determine Kotlin compiler in not hardcoded way
+    set(Kotlin_BIN_PATH
+            $ENV{HOME}/.konan/kotlin-native-macos-0.3/bin
```

Comment:
Probably you'd better use 0.6.2, as 0.3 is heavily obsolete.

Reply:
Yes I use 0.6.2. Dont know why it is here. And this two days I was writing special Lowering processor for call interception. Will submit PR this evening hopefully. Just for a review